manifests/d.pp

```diff
@@ -116,11 +123,15 @@
   # If both syslogging and mailing are requested, choose mailing over syslogging
   $actually_log_to_syslog = $log_to_syslog and $mailto=='""'
 
-  # Ancient versions of `timeout` have a slightly different argument syntax
-  # for what signal should be sent.
-  $timeout_signal_arg = "${::lsbdistid}_${::lsbdistrelease}" ? {
-    /(Ubuntu_10.04|CentOS_5.*)/ => '-9',
```

Comment:
We don't need to support lucid any more, so this logic can go away and we can just set -s 9